### DISCUSSION TITLE
Stratum 1 support

### DISCUSSION THREAD

**Discussion by tuncatunc** [coinfoundry/miningcore#939] [Q&A] [answered]
Hi All, 
The gpu miners can connect to eth pool over stratum2 ```stratum2+tcp://```
Litecoin pool users want to connect to using ```stratum+tcp://```
However they cannot. They say their asic device uses cgminer and it doesn't support stratum2.
My question is, is there a way miningcore support stratum?
Thx

**Comment by oliverw** [comment, marked as answer]
What has Litecoin to do with Ethereum?

**Reply by tuncatunc** [reply]
So the stratum is only for ethereum? We are unable to connect an ltc asic miner to the pool with stratum+tcp://... parameter, probably no need to specify stratum as it has nothing to do with ltc pool.

**Comment by oliverw** [comment]
@tuncatunc Are you a pool op or a miner?

**Reply by tuncatunc** [reply]
I want to operate a pool of btc ltc eth and doge

My client has an ASIC and he cannot connect to pool with

stratum+tcp://35.240.211.223:3042

They say they're unable to connect 

http://35.240.211.223:4000/api/pools/ltc1

Litecoin api

**Reply by tuncatunc** [reply]
My bad. Those ports were blocked by firewall
